client/src/game/carrier.js

```diff
@@ -63,12 +63,30 @@ class Carrier extends EventEmitter {
     this.drawGarrison()
   }
 
+  drawShape() {
+    this._rotateCarrierTowardsWaypoint(this.graphics_colour);
```

Comment:
Move this after the `switch` statement so that the shapes are not rotated.

This avoids the issue of diamonds looking like squares if they are rotated 90 degrees.